doc/syslog/syslog-design.md

```diff
@@ -430,10 +634,14 @@ _config syslog add_
 1. `-s|--source` - source ip address
```

Comment:
Are we planning to add config commands for SYSLOG_CONFIG too?

Reply:
As we discussed, define new CLI that can be added to sonic, but we won’t implement it as part of the push of the feature